Update README.md

Schwad opened the pull request:
For future users, this updates the forked gem documentation and links to be commensurate with the new maintained gem.

kylefox commented:
Hmm. I think the `Gemfile` needs an explicit `require`:

```ruby
gem 'trix-rails', require: 'trix'
```

Also, I haven't yet setup TravisCI, so that'll currently be a dead link. But it's something I hope to do soonish: https://github.com/kylefox/trix/issues/3

Commit:
Updates README.md
This PR implements changes to the README to update our references with the new community fork of the gem.

Schwad commented:
Thanks for taking a look. I reverted the travis-ci changes pending #3 , updated the require line, and squashed the commits into one.

kylefox commented:
Thank you!
